### ISSUE TITLE
removeaccess fails

### ISSUE THREAD

**Issue by Jaxom99** [YunoHost/issues#1523]
Hi team :wave: 

On my setup (Scaleway VPS, last version of yunohost, 4 apps, 42 users), I cannot remove access to apps using ``removeaccess`` by CLI or web interface. There is no error message, but the operation is not fulfilled and the user still has access. I tried by first using 'add access to all users" button, or just adding 3 users and then removing one : it behaves the same. But I can still reset, using the "erase all access rules" button.

The only clue I see is that the "authorized users" list is empty on return (see log), but still full when you get back on the app access settings. The _addaccess_ operation is logged, but not the _removeaccess_ one.

Any help is welcome !

My log : 
```
root@scw-metanum:/var/log/yunohost/categories/operation# yunohost  app  removeaccess etherpad_mypads -u bteitgen --debug
156  DEBUG loading actions map namespace 'yunohost'
177  DEBUG extra parameter classes loaded: ['comment', 'ask', 'password', 'required', 'pattern']
178  DEBUG initializing base actions map parser for cli
179  DEBUG registering new callback action 'yunohost.utils.packages.ynh_packages_version' to ['-v', '--version']
275  DEBUG lock has been acquired
315  DEBUG loading python module yunohost.app took 0.040s
315  INFO processing action [7499.1]: yunohost.app.removeaccess with args={'apps': ['etherpad_mypads'], 'users': ['bteitgen']}
372  DEBUG initialize authenticator 'as-root' with: uri='ldapi://%2Fvar%2Frun%2Fslapd%2Fldapi', base_dn='dc=yunohost,dc=org', user_rdn='gidNumber=0+uidNumber=0,cn=peercred,cn=external,cn=auth'
1880 SUCCESS The SSOwat configuration has been generated
1880 DEBUG action [7499.1] executed in 1.564s
1880 DEBUG lock has been released
allowed_users: 
```

**Comment by zamentur** [comment]
Which yunohost version are you using ?

**Comment by Jaxom99** [comment]
Latest (as far as I can tell), so 3.6.5.3
```
admin@scw-metanum:~$ sudo yunohost --version
yunohost: 
  repo: stable
  version: 3.6.5.3
yunohost-admin: 
  repo: stable
  version: 3.6.5.1
moulinette: 
  repo: stable
  version: 3.6.4.1
ssowat: 
  repo: stable
  version: 3.6.4
```

**Comment by Jaxom99** [comment]
Anything I can add to help ?

**Comment by alexAubin** [comment]
Uh I guess it's clearly a bug, but I don't know what to do about this since we're completely reworking the permission system for 3.7 and this command will disappeared and be replaced by something different ...

**Comment by Jaxom99** [comment]
Right, ofc, I didn't have that view of the project. Thanks for letting me know, I'll wait for 3.7 then ! 😄

**Comment by alexAubin** [comment]
Closed because 3.7 got release
